### PR TITLE
Radiology insights ga

### PR DESCRIPTION
**Pull request by koen-mertens**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsAsyncClient.java [hunk 1]
```diff
@@ -272,160 +283,299 @@ public final class RadiologyInsightsAsyncClient {
...
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public PollerFlux<BinaryData, BinaryData> beginInferRadiologyInsights(BinaryData radiologyInsightsData,
+    public PollerFlux<BinaryData, BinaryData> beginInferRadiologyInsights(String id, BinaryData resource,
         RequestOptions requestOptions) {
-        return this.serviceClient.beginInferRadiologyInsightsAsync(radiologyInsightsData, requestOptions);
+        return this.serviceClient.beginInferRadiologyInsightsAsync(id, resource, requestOptions);
+    }
+
+    @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
+    public PollerFlux<RadiologyInsightsData, RadiologyInsightsInferenceResult> beginInferRadiologyInsights(String id,
```
Comment (weidongxu-microsoft) [inline sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsAsyncClient.java:541]: need javadoc on this method

Comment (koen-mertens) [reply sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsAsyncClient.java:541]: added one

File: sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsAsyncClient.java [hunk 2]
```diff
@@ -272,160 +283,315 @@ public final class RadiologyInsightsAsyncClient {
...
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return the {@link PollerFlux} for polling of provides status details for long running operations.
+     * @return the {@link PollerFlux} for polling of response for the Radiology Insights request.
+     */
+    @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
+    public PollerFlux<RadiologyInsightsData, RadiologyInsightsInferenceResult> beginInferRadiologyInsights(String id,
+        RadiologyInsightsData resource, List<String> expand) {
+        // Generated convenience method for beginInferRadiologyInsightsWithModel
```
Comment (haolingdong-msft) [inline sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsAsyncClient.java:559]: If it is not generated convenient method, better to remove this comment.

File: sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsAsyncClient.java [hunk 3]
```diff
@@ -272,160 +283,315 @@ public final class RadiologyInsightsAsyncClient {
...
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the {@link PollerFlux} for polling of response for the Radiology Insights request.
      */
-    @Generated
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public PollerFlux<RadiologyInsightsResult, RadiologyInsightsInferenceResult>
-        beginInferRadiologyInsights(RadiologyInsightsData radiologyInsightsData) {
+    public PollerFlux<RadiologyInsightsData, RadiologyInsightsInferenceResult> beginInferRadiologyInsights(String id,
+        RadiologyInsightsData resource) {
         // Generated convenience method for beginInferRadiologyInsightsWithModel
```
Comment (haolingdong-msft) [inline sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsAsyncClient.java:591]: same here, if it is not generated method, but customized one, better to remove the comment.
btw, could you please let me know what is the customization code change you have on this method?

File: eng/versioning/version_client.txt [hunk 4]
```diff
@@ -129,7 +129,7 @@ com.azure:azure-e2e;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-health-deidentification;1.0.0;1.0.0-beta.2
 com.azure:azure-health-insights-clinicalmatching;1.0.0-beta.1;1.0.0-beta.2
 com.azure:azure-health-insights-cancerprofiling;1.0.0-beta.1;1.0.0-beta.2
-com.azure:azure-health-insights-radiologyinsights;1.0.0-beta.1;1.0.0-beta.2
+com.azure:azure-health-insights-radiologyinsights;1.0.0-beta.2;1.0.0
```
Comment (XiaofeiCao) [inline eng/versioning/version_client.txt:132]:
```suggestion
com.azure:azure-health-insights-radiologyinsights;1.0.0-beta.1;1.0.0
```

Comment (koen-mertens) [reply eng/versioning/version_client.txt:132]: changed

File: sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsAsyncClient.java (hunk 3 above)
Comment (koen-mertens) [reply sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsAsyncClient.java:591]: removed comments

**Review by srnagar — APPROVED** [review]

**Comment by haolingdong-msft** [comment]
Hi @koen-mertens , would you check @weidongxu-microsoft's [pr](https://github.com/Azure/azure-sdk-for-java/pull/41862) to see if you want to apply the changes from his pr?

File: sdk/healthinsights/azure-health-insights-radiologyinsights/README.md [hunk 5]
```diff
@@ -25,51 +25,58 @@ For more information about creating the resource or how to get the location and
...
 ```
 
-#### Create a RadiologyInsightsClient with an API Key Credential
+#### Authenticate and create a RadiologyInsightsClient with DefaultAzureCredential
 
-Once you have the value for the API key, you can pass it as a string into an instance of **AzureKeyCredential**. Use the key as the credential parameter
-to authenticate the client. You may choose to build a **synchronous** or **asynchronous** client.
+This sample code shows how to to use DefaultAzureCredential to authenticate with the service. More documentation about this authentication method can be found [here][azure_credential].
+
+```java com.azure.health.insights.radiologyinsights.defaultazurecredential
```
[the remark this answers is not in the document]
Comment (koen-mertens) [reply sdk/healthinsights/azure-health-insights-radiologyinsights/README.md:48]: that is how it was set up so think we are all good here

**Pull request merged**